templates/zerver/integrations/errbot.md

```diff
@@ -25,23 +27,42 @@ Run your favorite chatbot in Zulip!
         BOT_LOG_FILE = r'<path/to/your/errbot/logfile.log>'
         BOT_LOG_LEVEL = logging.INFO
 
-        BOT_IDENTITY = {  # Fill this with the corresponding values in your bot's `.zuliprc`
-          'email': '<err-bot@your.zulip.server>',
+        BOT_IDENTITY = {
+          'email': '<your-bot-name@yourZulipDomain.zulipchat.com>',
           'key': '<abcdefghijklmnopqrstuvwxyz123456>',
-          'site': '<http://your.zulip.server>'
+          'site': '<{{ zulip_url }}>'
```

Comment:
I think these should be consistent with the rest of the place holders where we kind of spell things out. So...

```
        BOT_IDENTITY = {
          'email': '<your-bot@email.address>',
          'key': '<your-bot-api-key>',
          'site': '<your-zulip-organization-url>',
```